### PR TITLE
Update tests to core 6.1.2

### PR DESCRIPTION
**Pull request by andreasohlund**
Replaces https://github.com/Particular/NServiceBus.Gateway/pull/64

@Particular/nservicebus-maintainers please review

### CODE REVIEW

File: src/NServiceBus.Gateway.Tests/packages.config [hunk 1]
```diff
@@ -4,6 +4,6 @@
   <package id="ApprovalTests" version="3.0.11" targetFramework="net452" />
   <package id="ApprovalUtilities" version="3.0.11" targetFramework="net452" />
   <package id="Mono.Cecil" version="0.9.6.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.0" targetFramework="net452" />
```
Comment (timbussmann) [inline src/NServiceBus.Gateway.Tests/packages.config]: should the unit tests also reference 6.1.2 (as they reference the gateway directly which is still targeting 6.0.0)?

Comment (andreasohlund) [inline src/NServiceBus.Gateway.Tests/packages.config]: I didn't put much thought into it, do you see any issues?

Comment (danielmarbach) [inline src/NServiceBus.Gateway.Tests/packages.config]: Have you looked into the output folder to check what version of NSB will be used in the tests?

Comment (timbussmann) [inline src/NServiceBus.Gateway.Tests/packages.config]: > do you see any issues?

not directly, it's the same with the acceptance tests anyway so I guess it doesn't really matter.

**Comment by andreasohlund** [comment]
The bin folders of the tests all contains 6.1.2 so I think we're good?

**Comment by SimonCropp** [comment]
@Particular/nservicebus-maintainers you still working on this?

**Comment by andreasohlund** [comment]
I'll re-update once on the new csproj format. Please go ahead with your stuff @SimonCropp

**Pull request closed without merging**